libretroshare/src/gxs/rsgxsnettunnel.cc

```diff
@@ -1155,22 +1155,7 @@ void RsGxsNetTunnelService::getStatistics(std::map<RsGxsGroupId,RsGxsNetTunnelGr
     bias = mRandomBias ;
 }
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+bool RsGxsNetTunnelService::isGXSHash(RsFileHash hash) const
```

Comment:
misleading because this is only valid for local hashes.

Reply:
I don't understand. This is wrong why?